### PR TITLE
Set recording episode name to correct place

### PR DESCRIPTION
**Pull request by hackthis02**
The episode name was being set to strPlotOutline not strEpisodeName

### CODE REVIEW

File: src/pvrclient-mediaportal.cpp [hunk 1]
```diff
@@ -969,8 +969,8 @@ PVR_ERROR cPVRClientMediaPortal::GetRecordings(ADDON_HANDLE handle)
       strEpisodeName = recording.EpisodeName();
 
       PVR_STRCPY(tag.strRecordingId, strRecordingId.c_str());
+	  PVR_STRCPY(tag.strEpisodeName, recording.EpisodeName());
```
Comment (ksooo) [inline src/pvrclient-mediaportal.cpp]: Could you please fix indentation. thx.

**Review by ksooo — COMMENTED** [review]
Could you plaese add changelog update and addon version bump.

**Comment by margro** [comment]
Changes look good to me.

The episode name field did not exist in the past, therefore I've used the strPlotOutline field. I'm not sure if all skins are already using/showing the strEpisodeName field. Last time I checked, Confluence did not.

Since the addon already uses v1.4.7 due to my Android bugfix from yesterday, I will cherry-pick your changes and include them in v1.4.8.

**Comment by hackthis02** [comment]
I guessed it was a fix from an older version. Confluence and Estuary both use $INFO[ListItem.EpisodeName] in the PVR sections. I haven't been able to dig around yet and see if strEpisodeName equates to ListItem.EpisodeName. But since that was the only var for it, I thought it would be a good start.

**Comment by hackthis02** [comment]
I went ahead and merged the two branches on my side if it makes it easier for you.

**Comment by margro** [comment]
Merged via PR #51. Thanks !

**Pull request closed without merging**